lib/source/membership.rb

```diff
@@ -29,6 +29,9 @@ def merged_with(csv)
 
         pr = reconciler.reconciliation_data rescue abort($!.to_s)
         pr.each { |r| id_map[r[:id]] = r[:uuid] }
+      else
```

Comment:
> This else bugs me a little (as always).

Interesting! Is that just because `else` statements increase the complexity of the method? Or is it a more philosophical objection?